index.js

```diff
@@ -40,6 +41,7 @@ module.exports = function createStore (reducer, initialState) {
     dispatch: dispatch,
     subscribe: subscribe,
     getState: getState,
-    replaceReducer: replaceReducer
+    replaceReducer: replaceReducer,
+    combineReducers: combineReducers
```

Comment:
yep, keen. :+1: even better if you want to require some but not all the functions (for example, if you didn't use `combineReducers` so didn't want to include it in your bundle)